tools/serverpod_cli/lib/src/generator/dart/library_generators/repository_classes.dart

```diff
@@ -0,0 +1,800 @@
+import 'package:code_builder/code_builder.dart';
+import 'package:serverpod_cli/src/analyzer/entities/definitions.dart';
+import 'package:serverpod_cli/src/config/config.dart';
+import 'package:super_string/super_string.dart';
+
+class BuildRepositoryClass {
```

Comment:
Suggestin: Place this in a subfolder.

Nice to get the separation here going! 🥇 
Think it makes sense to place these in a subfolder since we don't build them directly.

Reply:
agreed